typescript.md

```diff
@@ -239,3 +239,30 @@ from JS).
   * enum should be available from JavaScript.
 
 [&#8593; back to TOC](#table-of-contents)
+
+* For enum values use `UPPER_CASE` style.
+
+  **Good:**
+
+  ```ts
+  enum Foo {
+      BAR,
+      BAZ
+  }
+
+  enum Bar {
+      FOO = 'FOO',
```

Comment:
I think this example should be clarified that string values can have any style. In fact, enums with string values have meaning if compiled code, which is used from JavaScript. But there strings in lowercase are used  very often, e.g. 

```js
new winston.Logger({level: 'error'}); // not 'ERROR'
```